uc-crux-llvm/uc-crux-llvm.cabal

```diff
@@ -12,9 +12,15 @@ Synopsis:      A bugfinding tool for C programs, using under-constrained symboli
 
 common bldflags
   ghc-options: -Wall
+               -Werror
```

Comment:
I'm not super keen on blanket `-Werror` (I think the specific ones below seem fine, though I don't recognize missing home modules), which often makes things really exciting around the time when we need to add support for a new ghc version (or experiment with one to avoid compiler bugs)

Reply:
It sounds like the concern is that `-Wall` combined with `-Werror` can lead to new build failures if a warning is added to `-Wall` between GHC versions - is that right? Would it address that concern if I inlined the set of warnings that `-Wall` consists of?

Reply:
That is one possibility (at least picking the ones that we are actually worried about). Another is to do something like what macaw does and have a separate cabal.project file for ci https://github.com/GaloisInc/macaw/blob/master/.github/workflows/ci.yaml#L21

Reply:
Ah, that seems like a good approach! I'll add a `cabal.project` file like that to be used in CI.

Reply:
Okay, since I'm lazy I just removed `-Werror` for now, might revisit in the future and add the `cabal.project` file, though that would probably be best done over the whole repo.